### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by retroboydev**

### CODE REVIEW

File: .storybook/customTheme.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { create } from '@storybook/theming';
+
+const themeOptions = {
+  base: 'dark',
+  brandTitle: 'JediSwap Profile Page Storybook',
```
Comment (retroboydev) [inline .storybook/customTheme.js:5]: please update the title

File: __mocks__/mockBe.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { guildsListWithScore as guildsList } from '../src/features/guilds/GuildsList/GuildsList.testData';
+import { defaultLeaderboardData as leaderboardData } from '../src/features/guilds/Leaderboard/Leaderboard.testData';
+import { defaultNftListItems as nftListItems } from '../src/features/nft/NftCarousel/NftCarousel.testData';
+import { defaultUserData } from '../src/features/profile/ProfileCard/ProfileCard.testData';
+import { defaultCurrenciesList } from '../src/features/wallet/WalletBalance/WalletBalance.testData';
+
+const ARTIFICIAL_DELAY_MS = 600;
+
+const handlers = [
+  graphql.query('GetActivities', (req, res, ctx) => res(
```
Comment (retroboydev) [inline __mocks__/mockBe.js:13]: don't forget to update all these mock handlers

File: package.json [hunk 3]
```diff
@@ -0,0 +1,160 @@
+{
+  "name": "jediswap-personal-profile",
```
Comment (retroboydev) [inline package.json:2]: please update

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <!-- Primary Meta Tags -->
+  <title>JediSwap Contributor Profile</title>
```
Comment (retroboydev) [inline public/index.html:6]: please update this and all the lines down to #22

File: public/locales/en/translation.json [hunk 5]
```diff
@@ -0,0 +1,139 @@
+{
+  "header": {
+    "connectWallet": "Connect wallet"
+  },
+  "contributorPage": {
+    "titles": {
+      "title": "JediSwap Contributor Profile"
```
Comment (retroboydev) [inline public/locales/en/translation.json:7]: don't forget to clean these files as well

File: src/app/store.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import activitiesReducer from '../features/activities/activitiesSlice';
+import walletReducer from '../features/wallet/walletSlice';
+import guidsReducer from '../features/guilds/guildsSlice';
+import profileReducer from '../features/profile/profileSlice';
```
Comment (retroboydev) [inline src/app/store.js:6]: don't forget to get rid of all this

File: src/components/Header/Header.jsx [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+  const { chainId } = useStarknetReact();
+  const location = useLocation();
+
+  return (
+    <HeaderContainer py={1}>
+      <Grid container columnSpacing={{ md: 2 }} alignItems="center">
+        <Grid item xs={6} sx={{ display: 'flex', alignItems: 'center' }}>
+          <HeaderLogo to="/home">
+            <div>
+              <svg width="88" height="20" viewBox="0 0 88 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (retroboydev) [inline src/components/Header/Header.jsx:36]: please save this svg as a file and include it like here, for example - https://github.com/jediswaplabs/personal-profile/blob/master/src/components/Header/Header.jsx#L17

File: src/components/InfoCard/InfoCard.jsx [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+import { InfoLabel, InfoBox } from './InfoCard.styles';
+import icon from '../../resources/images/infoicon.png';
+
+const InfoCard = () => {
+
+  return (
+    <InfoBox>
+      <Box style={{textAlign: 'end', paddingRight: '10px', paddingTop: '10px'}}>
+        <img src={icon} />
```
Comment (retroboydev) [inline src/components/InfoCard/InfoCard.jsx:19]: please save it as svg and use SvgIcon component like here - https://github.com/jediswaplabs/personal-profile/blob/master/src/components/Header/Header.jsx#L75
Here's the doc - https://mui.com/material-ui/icons/#svgicon

File: src/components/MintCard/MintCard.jsx [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+import { QuestBox, QuestCardTitle, QuestCardType, QuestCardDescription, QuestCardAddress, QuestCardBtn} from '../QuestCard/QuestCard.styles';
+import { getShortenAddress } from '../../common/addressHelper';
+import GradientButton from '../GradientButton/GradientButton';
+import claimed from '../../resources/images/claimed.png';
+
+const MintCard = ({ title, description, address, nftImg }) => {
+  const [isClaimed, setIsClaimed] = useState(false);
+
+  return (<QuestBox style={{ width: '700px', padding: '30px', marginTop: '20px', display: 'flex', justifyContent: 'space-between' }}>
```
Comment (retroboydev) [inline src/components/MintCard/MintCard.jsx:19]: why do you use the QuestBox item if you redefine all the styles?

File: src/components/MintCard/MintCard.jsx [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+      </QuestCardDescription>
+      {!isClaimed &&
+        <QuestCardBtn onClick={() => setIsClaimed(!isClaimed)}>
+          Claim NFT
+        </QuestCardBtn>
+      }
+      {isClaimed &&
+        <QuestBox style={{ width: '314px', padding: '6px', display: 'flex', justifyContent: 'center', alignContent: 'center' }}>
+          <div style={{ marginRight: '5px' }}>
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```
Comment (retroboydev) [inline src/components/MintCard/MintCard.jsx:38]: same comment like above for others svgs

File: src/components/MintCard/MintCard.jsx [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+      <QuestCardDescription style={{maxWidth: '314px'}}>
+        {description}
+      </QuestCardDescription>
+      {!isClaimed &&
+        <QuestCardBtn onClick={() => setIsClaimed(!isClaimed)}>
+          Claim NFT
+        </QuestCardBtn>
+      }
+      {isClaimed &&
+        <QuestBox style={{ width: '314px', padding: '6px', display: 'flex', justifyContent: 'center', alignContent: 'center' }}>
```
Comment (retroboydev) [inline src/components/MintCard/MintCard.jsx:36]: why do you use the QuestBox item if you redefine all the styles?

File: src/features/nft/NftCarousel/NftCarousel.jsx [hunk 12]
```diff
@@ -0,0 +1,97 @@
+/* eslint-disable import/no-unresolved */
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
```
Comment (retroboydev) [inline src/features/nft/NftCarousel/NftCarousel.jsx:3]: don't forget to get rid of all the features you don't need to

**Review by retroboydev — COMMENTED** [review]
let's discuss other comments in person

**Comment by retroboydev** [comment]
@iamoskvin ⬆️

**Comment by vnaysngh-mudrex** [comment]
not required anymore

**Pull request closed without merging**